keycloak/realm/realm-export.json

```diff
@@ -2336,7 +2336,7 @@
     "actionTokenGeneratedByUserLifespan-execute-actions": "",
     "actionTokenGeneratedByUserLifespan-verify-email": "",
     "clientOfflineSessionIdleTimeout": "0",
-    "userProfileEnabled": "true",
+    "userProfileEnabled": "false",
```

Comment:
userProfile is managed in self-serve-portal, so we could switch this to off, to make sure it is using default register.ftl template